Automattic/newspack-newsletters
# Update editor and MJML style

### All Submissions:






### Changes proposed in this Pull Request:

This brings the editor closer to what the email will actually look like. There are still a few differences (e.g. padding on blocks with background colour) but these are pretty minor differences.

### How to test the changes in this Pull Request:

1. Create a new newsletter. Send a test email. Spot the differences.
2. Switch to this branch.
3. Refresh editor and send another test email. There should be less differences.

### Other information:
